setup.cfg

```diff
@@ -27,9 +27,11 @@ console_scripts =
    pbench-clear-results = pbench.cli.agent.commands.results.clear:main
    pbench-clear-tools = pbench.cli.agent.commands.tools.clear:main
    pbench-config = pbench.cli.getconf:main
+   pbench-copy-results = pbench.cli.agent.commands.results:main
    pbench-generate-token = pbench.cli.agent.commands.generate_token:main
    pbench-list-tools = pbench.cli.agent.commands.tools.list:main
    pbench-list-triggers = pbench.cli.agent.commands.triggers.list:main
+   pbench-move-results = pbench.cli.agent.commands.results:main
```

Comment:
Both of these commands need to be added to `agent/Makefile` and the original file removed.

Reply:
ok